ddlitlab2024/ml/training/train.py

```diff
@@ -147,7 +177,7 @@
     )
 
     # Load the learning rate scheduler state if a checkpoint is provided
-    if args.checkpoint is not None:
+    if args.checkpoint is not None and False:
```

Comment:
This will always result in `False`.
I assume you meant `if args.checkpoint is not None and args.checkpoint is not False` in which case I would just do `if args.checkpoint:`

But I don't really see when it could be False anyway, since it is defined as a `str` argument.

Reply:
It was just a hack to deactivate this code path. I will clean this up with another flag. Because loading the learning rate scheduler state when starting from a pretrained model is not desirable (as the end of the schedule was reached during pretrained). In contrast to that you want to resume the schedule if e.g. the training was interrupted.